core/domain/feedback_services.py

```diff
@@ -819,7 +819,8 @@ def enqueue_feedback_message_batch_email_task(user_id):
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
 
 
-def enqueue_feedback_message_instant_email_task(user_id, reference):
+def enqueue_feedback_message_instant_email_task(
+        user_id, reference):
```

Comment:
Why this change?  It seems like the arguments fit on one line with the definition without going past the 80 character column limit.

Reply:
Done